gef-ui/server/models/application.js

```diff
@@ -94,6 +94,7 @@ class Application {
     try {
       const application = await getApplication(id);
 
+      // TODO DTFS2-6628: this doesn't look right - what about if the user has access to all banks?
       if (application.bank.id !== user.bank.id && !user.roles.includes(ADMIN)) {
         return null;
       }
```

Comment:
@abhi-markan would be curious to get your thoughts here. This logic seems to say that you can view the gef application if either:
- your bank matches the application's bank, or
- you are an admin

but I thought it should be either:
- your bank matches the application's bank, or
- your bank is the special "All" bank (regardless of admin or not)

Do you agree with me or do you think I have misunderstood?

Reply:
This is outside the scope of the current ticket so I'm going to close this off and will investigate it separately